cub/cub/block/block_adjacent_difference.cuh

```diff
@@ -54,8 +54,8 @@ CUB_NAMESPACE_BEGIN
  *   the elements partitioned across a CUDA thread block. Because the binary
  *   operation could be noncommutative, there are two sets of methods.
  *   Methods named SubtractLeft subtract left element `i - 1` of input sequence
- *   from current element `i`. Methods named SubtractRight subtract current
- *   element `i` from the right one `i + 1`:
+ *   from current element `i`. Methods named SubtractRight subtract the right one `i + 1`
+ *   from the current element `i`:
```

Comment:
remark: thank you for fixing this! To summarize, the change makes the description symmetrical:
```
left:  subtract(current, left):  subtract left element `i - 1` of input sequence from current element `i`
right: subtract(current, right): the right element `i + 1` from the current one `i`
```

suggestion: to my understanding, prop-words are typically used after the word they replace:
```suggestion
 *   from current element `i`. Methods named SubtractRight subtract the right element `i + 1`
 *   from the current one `i`:
```